modules/optimization/doc/content/modules/optimization/sqa/optimization_srs.md

```diff
@@ -1 +1,18 @@
-!template load file=app_srs.md.template app=Optimization category=optimization
+!template load file=sqa/module_srs.md.template app=Optimization category=optimization
+
+!template! item key=system-purpose
+!! system-purpose-begin
+The purpose of the MOOSE Optimization module optimize model parameters such that it minimizes a defined objective. An example would be to search for material property values so that the solution matches experimental results. As such, the purpose of this module is not to provide physical model capabilities, which is typically the responsibility of other MOOSE modules and dependent applications.
```

Comment:
The purpose of the MOOSE Optimization module is to control model parameters

Reply:
This should be improved in terms of grammar anyway - will leave it to you and Zach to come to a suitable solution on specific wording.